aws/resource_aws_emr_instance_fleet_test.go

```diff
@@ -0,0 +1,551 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAWSEMRInstanceFleet_basic(t *testing.T) {
+	var fleet emr.InstanceFleet
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resource.Test(t, resource.TestCase{
```

Comment:
Unless there is a need for serialized testing (e.g. only one can exist at a time in the region), we typically prefer `resource.ParallelTest()`

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```